### DISCUSSION TITLE
Sendmail differences ESP8266 ESP32

### DISCUSSION THREAD

**Discussion by Bebuccio** [arendst/Tasmota#18062] [Q&A] [answered]
I have used a customized version of "Tasmota 12.4.0 Peter" (#define USE_SENDMAIL).
With this version there is a difference between **ESP32** and **ES8266** version:
Using the version for ESP32 the email address must be written without the <>.
Using the ESP8266 version both email address (sender and receiver) **MUST** be surrounded by <>.

I.E. :
  for **ESP32** email address --> _user@server.com_
  for **ESP8266** email address --> _**<**user@server.com**>**_

For new releases it is possible to make the two versions working in the same manner?

**Comment by barbudor** [comment, marked as answer]
I'm looking which one would be the simplest
Changing the ESP8266 one to support emails without "<" seems better as it is in Tasmota code and not in library code

**Reply by barbudor** [reply]
Done: https://github.com/arendst/Tasmota/pull/18064
After the merge, both version will accept emails without "<" and ">"
